packages/x-license/src/utils/licenseErrorMessageUtils.ts

```diff
@@ -32,6 +32,16 @@ export function showLicenseKeyPlanMismatchError() {
   ]);
 }
 
+export function showProductScopeMismatchError() {
+  showError([
+    'MUI X: Invalid Product coverage.',
+    '',
+    'Your use of MUI X is not compatible with the plan of your license key. The product you are trying to use is not included in the version of your license plan. This happens if you try to use `ChartsPro` or TreeViewPro` with a license plan where these products are not included.',
+    '',
+    'To solve the issue, you can upgrade your plan version at https://mui.com/r/x-get-license.',
```

Comment:
```Suggestion
    `Please consider upgrading your plan version at https://mui.com/r/x-get-license.',
```

Reply:
I'm also thinking to instead of linking to "x-get-license" linking users to an url specific about the upgrading process, which, as of now, involves contacting the sales channel.

```Suggestion
    'Please consider upgrading your plan version. You may be entitled to a discount. More information at https://mui.com/r/x-upgrade-from-pro-legacy',
```

Reply:
does the site exist already?

Reply:
No, we'd have to create the redirect address and the link target.

Reply:
OK, so for now I would either redirect to `https://mui.com/r/x-get-license` or simply adding `sales@mui.com` .. WDYT?

Reply:
I removed the link completely for now and went with `'To start using another Pro package, please consider reaching to our sales team to upgrade your license.'` from the change request above

Reply:
I suggested adding the link because we can update the link independently later, and for now the link could also redirect to the same address of: `https://mui.com/r/x-get-license`